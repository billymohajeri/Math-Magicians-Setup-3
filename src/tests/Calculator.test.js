import React from 'react';
import renderer from 'react-test-renderer'
import Calc from '../components/Calculator';

it('renders correctly', () => {
  const calculator = renderer
    .create(<Calc />)
    .toJSON();
  expect(calculator).toMatchSnapshot();
});