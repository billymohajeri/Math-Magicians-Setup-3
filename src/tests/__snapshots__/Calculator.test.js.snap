// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main-calc"
>
  <h2
    className="calc-title"
  >
    Let's do some math!
  </h2>
  <div
    className="calc"
  >
    <div
      className="first row"
    >
      <p
        className="result"
      />
    </div>
    <div
      className="second row"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="third row"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="fourth row"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="fifth row"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="sixth row"
    >
      <button
        className="double"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
